upb_generator/protoc-gen-upbdefs.cc

```diff
@@ -62,7 +66,8 @@ void GenerateMessageDefAccessor(upb::MessageDefPtr d, Output& output) {
   output("\n");
 }
 
-void WriteDefHeader(upb::FileDefPtr file, Output& output) {
+void WriteDefHeader(upb::FileDefPtr file, Options const& options,
```

Comment:
west const please (applies throughout the file)

Reply:
> west const please (applies throughout the file)

@fowles This is now fixed